Tests/XCUITests/HistoryTests.swift

```diff
@@ -103,215 +98,214 @@ class HistoryTests: BaseTestCase {
-
-        app.tables.cells.staticTexts[closedWebPageLabel].press(forDuration: 1)
+        XCTAssertTrue(app.tables.cells.staticTexts[bookOfMozilla["label"]!].exists)
+        app.tables.cells.staticTexts[bookOfMozilla["label"]!].press(forDuration: 1)
         waitForExistence(app.tables["Context Menu"])
         app.tables.otherElements[ImageIdentifiers.newPrivateTab].tap()
+        app.buttons["Done"].tap()
 
-        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        // The page is opened only on the new private tab
+        navigator.nowAt(HomePanelsScreen)
         navigator.goto(TabTray)
-        let numTabsOpen = userState.numTabs
-        XCTAssertEqual(numTabsOpen, 1)
+        XCTAssertFalse(app.staticTexts[bookOfMozilla["title"]!].exists)
+        navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
+        XCTAssertTrue(app.staticTexts[bookOfMozilla["title"]!].exists)
+        XCTAssertEqual(userState.numTabs, 1)
     }
 
     func testPrivateClosedSiteDoesNotAppearOnRecentlyClosed() {
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
-        waitForTabsButton()
+        // Open "Book of Mozilla" page in regular mode (not private mode)
+        openBookOfMozilla()
+
+        // Open the two tabs in private mode
+        // It is necessary to open two sites. When one tab is closed private mode, the private mode still has something opened.
         navigator.toggleOn(userState.isPrivate, withAction: Action.TogglePrivateMode)
-        // Open the default website
-        userState.url = path(forTestPage: "test-mozilla-book.html")
-        navigator.nowAt(TabTray)
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.openURL(userState.url!)
-        // It is necessary to open two sites so that when one is closed private mode is not closed
+        navigator.openURL(path(forTestPage: bookOfMozilla["file"]!))
+        waitUntilPageLoad()
+        navigator.nowAt(BrowserTab)
         navigator.openNewURL(urlString: path(forTestPage: "test-mozilla-org.html"))
         waitUntilPageLoad()
-        waitForTabsButton()
+        navigator.nowAt(BrowserTab)
+
+        // Close the private tab "Book of Mozilla" by tapping 'x' button
         navigator.goto(TabTray)
         waitForExistence(app.cells.staticTexts[webpage["label"]!])
-        // Close tab by tapping on its 'x' button
         if isTablet {
             app.otherElements["Tabs Tray"].collectionViews.cells.element(boundBy: 0).buttons["tab close"].tap()
         } else {
             app.otherElements.cells.element(boundBy: 0).buttons["tab close"].tap()
         }
 
-        navigator.performAction(Action.OpenNewTabFromTabTray)
-        navigator.goto(HomePanelsScreen)
-        navigator.performAction(Action.CloseURLBarOpen)
-        navigator.nowAt(NewTabScreen)
+        // On private mode the "Recently Closed Tabs List" is empty
```

Comment:
Trying to go to LibraryPanel_History as the test is now passes by the HomePanelScreen and if so it is necessary to dismiss the keyboard as in previous examples